### PR TITLE
Backport #279 to main-0.4

### PR DESCRIPTION
**Pull request by matthme**

### CODE REVIEW

File: .github/workflows/bundling-test.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
   push:
     branches:
-      - "main"
+      - "main-0.4"
```
Comment (jost-s) [inline .github/workflows/bundling-test.yml]: This is only for pushes, and generally pushing to main is a rare exception.

File: .github/workflows/integration-test.yml [hunk 2]
```diff
@@ -4,8 +4,7 @@ on:
   pull_request:
   push:
     branches:
-      - "develop"
-      - "main"
+      - "main-0.4"
```
Comment (jost-s) [inline .github/workflows/integration-test.yml]: Same here. We don't need `develop` here anymore, just `main`.

**Review by jost-s — CHANGES_REQUESTED** [review]
LGTM, just the workflow changes
and please update the changelog

File: .github/workflows/bundling-test.yml (hunk 1 above)
Comment (matthme) [reply .github/workflows/bundling-test.yml]: Oh, isn't it for pull requests too? I just wanted the tests to run on this PR (which is to main-0.4) to make sure I didn't mess up with my backporting.

**Comment by matthme** [comment]
Okay, requested changes should be addressed.

File: .github/workflows/bundling-test.yml (hunk 1 above)
Comment (jost-s) [reply .github/workflows/bundling-test.yml]: There's nothing after `pull_request:`, which means it's not restricted. `push:` is limited to branches, branches that are called `main`. So it should run on all PRs anyway.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,12 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
 
 ### Added
+- Bring back a websocket reconnection automation for Admin and App websockets. When either of them is closed and a new request made, it will attempt to reconnect using the same app authentication token that was used to initially authenticate the websocket. A specific `InvalidTokenError` is returned if that fails.
 ### Changed
 - Update to Holochain 0.4.0-rc.0
 ### Fixed
 ### Removed
 
-## 2024-09-30: v0.18.0-dev.12
+## 2024-09-30: v0.18.0-dev.13
```
Comment (jost-s) [inline CHANGELOG.md:14]: :+1:

**Review by jost-s — APPROVED** [review]
Great!

**Pull request merged**